### DISCUSSION TITLE
Function `update_script` not called

### DISCUSSION THREAD

**Discussion by Zelnes** [tteck/Proxmox#2656] [Help Needed] [answered]
I was trying to install the wireguard container ([this one](https://github.com/tteck/Proxmox/blob/main/ct/wireguard.sh)), but the dashboard was never installed.
I tried to understand the scripts, but it seems the function `update_script`, which in wireguard's case is the one containing the installation steps, is not called.
I compared it to technitium ([this one](https://github.com/tteck/Proxmox/blob/main/ct/technitiumdns.sh)), but didn't find the code difference. But this one worked perfectly on the first run.

I finally managed to get the dashboard to work, but I had to manually run the `update_script` code.

Also, I've seen that the installation end message `"${APP} should be reachable by going to the following URL"` is in the global context for technitium, but in the `update_script` function for wireguard.

When is this function supposed to be called ?

**Comment by tteck** [comment, marked as answer]
When you type the command `update` in the LXC console.

**Reply by tteck** [reply]
![Screenshot 2024-03-13 9 09 57 PM](https://github.com/tteck/Proxmox/assets/21342670/a6d29bf6-324e-496f-b219-45295d677c1c)

**Reply by Zelnes** [reply]
Ohhhh, I'm sorry, I didn't think the website would held more information than the repository.
I did all my research through the github.
Thanks !
